### ISSUE TITLE
Fix the mobile view

### ISSUE THREAD

**Issue by Dhravya** [Dhravya/blog#28]
The front page of the blog is not mobile-compatible. It breaks after a certain width. 
![demonstration](https://us-east-1.tixte.net/uploads/img.dhravya.dev/l3159695k0a.gif)

I'll mark this as a good first issue if anyone else wants to collaborate

**Comment by Ho3pLi** [comment]
where i can find the code?

**Comment by Dhravya** [comment]
hey, you can go to the index.jsx page where the main page is rendered.im using CSS in JS so it should be really straightforward. just go through the source folder

**Comment by Ho3pLi** [comment]
I work there as soon as I have time :)

**Comment by Dhravya** [comment]
Amazing,I'll assign it to you

**Comment by Ho3pLi** [comment]
I was wondering, how can I launch the site to apply changes locally? in short, for development purposes

**Comment by Dhravya** [comment]
you can run the command "npm run develop"

**Comment by Dhravya** [comment]
Yeah, it's a version issue. You can either force install `npm i --force` or just use yarn, which resolves everything automatically `yarn install`

**Comment by Dhravya** [comment]
I'm sorry for the late response 😓

**Comment by Ho3pLi** [comment]
> I'm sorry for the late response sweat

np dude, i'll try with yarn

**Comment by Ho3pLi** [comment]
dude there are SO MANY warnings ahahah
![image](https://user-images.githubusercontent.com/61387309/168024334-a74c02e9-3e1b-4c81-8c20-239f4a25cafb.png)
![image](https://user-images.githubusercontent.com/61387309/168024500-790bac81-da79-4d13-95f9-cc9713a592ac.png)

**Comment by Ho3pLi** [comment]
I literally just figured out how you integrate the css aaaaaaaaaaa 
ok now i can do something

**Comment by Dhravya** [comment]
Yes, the CSS is inside the `css={{ <CSS-in-JS> }}` 😅😅

**Comment by Ho3pLi** [comment]
oh isn't it? :sob: 
![image](https://user-images.githubusercontent.com/61387309/168034399-68046adc-9d27-4fb9-a9b2-084db4c9f634.png)

**Comment by Dhravya** [comment]
That CSS is for the Typography of the individual posts

**Comment by Dhravya** [comment]
Have a look at https://github.com/Dhravya/blog/blob/e796936c08a688b085c603abb71e7838de77852f/src/pages/index.jsx#L53-L67

**Comment by Ho3pLi** [comment]
this grid rappresents all the content in the page?

**Comment by Dhravya** [comment]
The grid has 2 parts- <Sidebar> and <Layout>, so that's probably what you have to work on

**Comment by Ho3pLi** [comment]
I couldn't do it I'm sorry :cry:

**Comment by Dhravya** [comment]
ah thats completely fine. no issues 😄

**Comment by Harjot4698** [comment]
hie @Dhravya , I m interested in you issue , reply when you get this message.

**Comment by Dhravya** [comment]
Sure! I'll assign it to you
